app/services/file_upload/file_uploader.rb

```diff
@@ -24,5 +24,9 @@ def scan_file(file)
 
       raise PotentialMalwareError unless result
     end
+
+    def self.human_readable_max_file_size
+      "#{(ENV['MAX_UPLOAD_SIZE_BYTES'].to_i / (1024 * 1024.0)).round}MB"
```

Comment:
either way
```rb
include ActionView::Helpers::NumberHelper
number_to_human_size(ENV['MAX_UPLOAD_SIZE_BYTES'].to_i)
=> "10 MB"
```

Reply:
Ach, frustratingly the convention our copy uses is to not have a space before "MB", and that space in non-configurable in NumberHelper. I could gsub it out, but at that point I think it's probably cleaner just to do it outselves.